README.md

```diff
@@ -4,10 +4,11 @@
 
 # HiveMQ MQTT Client
 
-[![Build Status](https://travis-ci.com/hivemq/hivemq-mqtt-client.svg?branch=develop)](https://travis-ci.com/hivemq/hivemq-mqtt-client)
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.hivemq/hivemq-mqtt-client/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.hivemq/hivemq-mqtt-client)
 [![JitPack](https://jitpack.io/v/hivemq/hivemq-mqtt-client.svg)](https://jitpack.io/#hivemq/hivemq-mqtt-client)
+[![Build Status](https://travis-ci.com/hivemq/hivemq-mqtt-client.svg?branch=develop)](https://travis-ci.com/hivemq/hivemq-mqtt-client)
```

Comment:
Why do you track the develop branch for the badge? I know you are merging into develop but on the master branch readme develop is tracked, too.

Reply:
good point, but it is not possible to have a different readme on the master branch.
And a released version on the master branch should always have a passing build.

Reply:
Yeah, but it would look pretty odd if the builds are failing or paused on the master branch even if they are completely fine just because the develop branch failed.

Reply:
The develop branch should also not fail, as we are only merging if all checks pass